scripts/obtain/dotnet-install.ps1

```diff
@@ -366,6 +367,29 @@ function DownloadFile([Uri]$Uri, [string]$OutPath) {
     }
 }
 
+function Get-Runtime-Version-From-Dotnet-Package([System.IO.Compression.ZipArchive]$Zip) {
```

Comment:
My concern there was that if there's a higher version installed than the consuming repo wants to use, we will use the higher version instead.  If that's not a problem, I'll change this to just use the highest version.